tests/PHPStan/Analyser/nsrt/bug-12902-non-strict.php

```diff
@@ -0,0 +1,88 @@
+<?php declare(strict_types = 0); // lint >= 8.1
```

Comment:
The first line has to be `<?php // lint >= 8.1` and the declare statement has to be on line 3.

Reply:
thanks!